vertx-web-graphql/pom.xml

```diff
@@ -62,6 +64,32 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-client</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-rx-java2</artifactId>
+    </dependency>
```

Comment:
This creates a circular dependency. I think we can do with reactive streams API only.

Reply:
Ok, I've removed that dependency that was used in SubscriptionIT which has been removed too.